examples/integrations/oci/base/identity-se/oci-setup.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+#
```

Comment:
Seems like this file is duplicate is several places, why not moving under examples/k8s ?

Reply:
Trying to keep examples self-sufficient. There is another example under cdi/oci-objectstorage. May be reused as well.